src/components/PayPalSDKWrapper.js

```diff
@@ -51,6 +51,7 @@ function PayPalSDKWrapper(props: Props) {
     } = config;
 
     const params = {
+      vault: true,
```

Comment:
I don't quite know if setting this to `true` per default will break something on upgrade for users of the `SmartPaymentButtons`. I'd rather set it to false and add to the README that it needs to be set to true to allow subscriptions (and the use of the `SmartSubscriptionButtons` respectively).